packages/bp/src/core/messaging/messaging-service.ts

```diff
@@ -138,16 +152,37 @@ export class MessagingService {
     }
 
     const payloadAbsoluteUrl = this.convertToAbsoluteUrls(event.payload)
+
+    const collecting = event.incomingEventId && this.collectingCache.get(event.incomingEventId)
     const message = await this.clientsByBotId[event.botId].messages.create(
       event.threadId!,
       undefined,
-      payloadAbsoluteUrl
+      payloadAbsoluteUrl,
+      collecting ? { incomingId: collecting } : undefined
     )
     event.messageId = message.id
 
     return next(undefined, true, false)
   }
 
+  public informProcessingDone(event: IO.IncomingEvent) {
+    if (this.collectingCache.get(event.id!)) {
+      // We don't want the waiting for the queue to be empty to freeze other messages
+      // eslint-disable-next-line @typescript-eslint/no-floating-promises
+      this.sendProcessingDone(event)
```

Comment:
```suggestion
      void this.sendProcessingDone(event)
```

Reply:
This was asked by yann to use the comment instead. I guess in this repo we don't use void